CK3toEU4/Source/CK3World/Dynasties/Dynasties.h

```diff
@@ -12,10 +13,12 @@ class Dynasties: commonItems::parser
 	explicit Dynasties(std::istream& theStream); // For testing
 
 	[[nodiscard]] const auto& getDynasties() const { return dynasties; }
+	[[nodiscard]] auto getHouses() { return std::move(houses); }
```

Comment:
For cases where you std::move an object, I prefer take to get. I'm also not fond of it outside temporary classes, and getDynasties not being the same implies this is meant to be a more permanant class.

Reply:
This is a feature of their writing formats. They bundle stuff often like this:
domain = {
domain = {}
houses = {}
}
and while the two are related (same as faiths/religions) they are really standalone classes that handle their own internals. So I shove one outside the other and world gets management of them independently. I'd rather move it and let it crash if I try to access houses within domain, than work over a copy.